draft-ietf-ppm-dap.md

```diff
@@ -1891,26 +1916,34 @@ The Leader proceeds as follows with each report:
    * `task_id` is the task ID
    * `agg_param` is the VDAF aggregation parameter provided by the Collector
      (see {{collect-flow}})
-   * `prev_state` is the state computed earlier by calling
-     `Vdaf.ping_pong_leader_init` or `Vdaf.ping_pong_leader_continued`
```

Comment:
Consistency: Elsewhere we write `state = Vdaf.ping_pong(state)` rather than `state = Vdaf.ping_pong(prev_state)`

Reply:
I don't mind renaming it from `prev_state`, but we should still describe the `state` value input to `Vdaf.ping_pong_leader_continued`

Reply:
Done.